src/app/dj/overview/page.tsx

```
@@ -0,0 +1,336 @@
+              </Link>
+            </Button>
+          </CardContent>
+        </Card>
+      )}
+
+      {score === 100 && (
+        <Card className="border-green-500/20 bg-green-500/5">
+          <CardContent className="flex items-center gap-2 py-4">
+            <CheckCircle2 className="h-5 w-5 text-green-400" />
+            <span className="text-sm font-medium text-green-300">
+              Your profile is 100% complete — you're ready to get booked!
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)